### PR TITLE
Merge FLIP Fest PRs

### PR DESCRIPTION
**Pull request by MaxStalker**
This PR shall resolve all merge conflicts and bring Playground back into stable form.

### CODE REVIEW

**Comment by MaxStalker** [comment]
@10thfloor can you tell what was the problem with our tutorials so I can check if this was fixed?

File: src/providers/Project/projectMutator.ts [hunk 1]
```diff
@@ -106,8 +124,8 @@ export default class ProjectMutator {
     } else {
       Mixpanel.track('Project saved', { projectId: this.projectId });
     }
-    
-    navigate(`/${this.projectId}`, { replace: true});
+
+    navigate(`/${this.projectId}?type=readme&id=0`, { replace: true });
```
Comment (psiemens) [inline src/providers/Project/projectMutator.ts:128]: I think the README should just be the default view when loading a project. Is `?type=readme` required?

File: src/util/url.ts [hunk 2]
```diff
@@ -11,6 +11,6 @@ export const getParams = (url: string): any =>  {
   },{})
 }
 
-export const scriptTypes = ["account", "tx", "script"]
+export const scriptTypes = ["account", "tx", "script", "readme"]
```
Comment (psiemens) [inline src/util/url.ts:14]: Should this be `pageTypes` instead of scripts?

**Review by psiemens — APPROVED** [review]
Looks good @MaxStalker!

I have a comment about the URL structure, but don't let it block this PR.

**Pull request merged**